lib/matplotlib/contour.py

```diff
@@ -1409,64 +1409,38 @@ def find_nearest_contour(self, x, y, indices=None, pixel=True):
 
         Returns
         -------
-        contour : `.Collection`
-            The contour that is closest to ``(x, y)``.
+        path : int
```

Comment:
The type hint should also be updated:

https://github.com/matplotlib/matplotlib/blob/39582424f8ce5e042e4e39eeac94893f222c9f8b/lib/matplotlib/contour.pyi#L162

(I trusted the docstring and unfortunately stubtest can't check return types... Inline type hints may have caught this particular error)

Reply:
Done.  I have to admit I've been relying on the CI for type hinting stuff.  Will try to Do Better in future.

Reply:
Yeah, the CI we have is good about catching newly added/removed methods and method input changes, but return types are harder to check without calling the method (and even then you have to know valid inputs). (And if it would have caught this, it probably would have caught it from the start)